### PR TITLE
contribute

### PR DESCRIPTION
**Pull request by millergarym**

### CODE REVIEW

**Comment by millergarym** [comment]
Moving pull request from pboyer/antlr4 to antlr/antlr4

This fixes an issue with generated Go code where a local var gets declared twice in the same scope.
Example grammar
```
grammar redeclared;

s : A=ID B=ID
  | X=id Y=id
  ;

id : ID2;

ID : 'a';
ID2 : 'b';
```

File: runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../../../antlr
```
Comment (pboyer) [inline runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr]: @millergarym - What is this?

Comment (millergarym) [inline runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr]: @pboyer - It is a soft link to the directory containing the Go runtime.
I had a test Go file that I used to test the generate code.
The runtime source needs to be under a `src` directory for `go build`
Git will recreate the link of Unix based system, not sure what happens on Windows.

Comment (parrt) [inline runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr]: @pboyer lemme know if/when this PR is acceptable.

Comment (parrt) [inline runtime/Go/src/github.com/antlr/antlr4/runtime/Go/antlr]: @pboyer  should I accept this merge?

**Pull request closed without merging**